scalahost/nsc/src/test/scala/scala/meta/tests/scalahost/mirrors/DatabaseSuite.scala

```diff
@@ -87,7 +87,7 @@ abstract class DatabaseSuite(mode: SemanticdbMode) extends FunSuite with DiffAss
     val path = g.currentRun.units.toList.last.source.file.file.getAbsolutePath
     val payload = database.toString.split(EOL)
     val section = payload.dropWhile(_ != sectionName + ":").drop(1).takeWhile(_ != "")
-    // println(section.mkString(EOL).replace(path, "<...>"))
+//     println(section.mkString(EOL).replace(path, "<...>"))
```

Comment:
Do we treat this PR as something that we plan to discuss or something that we plan to merge? If that's the latter, I suggest we don't leave comments like that in the diff.